### PR TITLE
{2023.06,a64fx}[foss/2023a] ParaView 5.11.2

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/a64fx

**Comment by boegel** [comment]
Hmm, something is fishy there...
```
ERROR: software.eessi.io/versions/2023.06/compat/linux/aarch64 does not exist!
```

**Comment by boegel** [comment]
I'm also seeing this:
```
repo 'eessi.io-2023.06-software' is not an EESSI CVMFS repository...
```

Looks like this is fallout from the changes made in #618... There's an `/cvmfs` missing somewhere?

@Neves-P Any ideas here?

**Comment by boegel** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/a64fx

**Comment by boegel** [comment]
testing fix from #633 in here...

**Comment by boegel** [comment]
Build of `Qt5` failed due to lack of sufficient memory:
```
g++: fatal error: Killed signal terminated program cc1plus
```

**Comment by boegel** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/a64fx

**Comment by boegel** [comment]
Job 32801 is still chewing on ParaView itself, all dependencies worked.

Creating tarball will fail though, we'll need to restart the build to pick up on the fix made in #635...

**Comment by boegel** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/a64fx

File: eb_hooks.py [hunk 1]
```diff
@@ -81,8 +81,11 @@ def post_ready_hook(self, *args, **kwargs):
     """
     # 'parallel' easyconfig parameter is set via EasyBlock.set_parallel in ready step based on available cores.
     # here we reduce parallellism to only use half of that for selected software,
-    # to avoid failing builds/tests due to out-of-memory problems
-    if self.name in ['TensorFlow', 'libxc']:
+    # to avoid failing builds/tests due to out-of-memory problems;
+    memory_hungry_build = self.name in ['libxc', 'TensorFlow']
+    # on A64FX systems, (HBM) memory is typically scarce, so we need to use fewer cores for some builds
+    memory_hungry_build_a64fx = self.name in ['Qt5']
```
Comment (bedroge) [inline eb_hooks.py:87]: Shouldn't this also check for the CPU target, as it now seems to apply this for all CPUs?

Comment (boegel) [reply eb_hooks.py]: Yes, of course... Fixed in b07445e

**Review by bedroge — APPROVED** [review]

**Comment by boegel** [comment]
Deploy triggered.

I've stopped my bot instance on Deucalion for now

**Comment by bedroge** [comment]
Ingested, and this is now available:
```
$ ls /cvmfs/software.eessi.io/versions/2023.06/software/linux/aarch64/a64fx/software/ParaView/
5.11.2-foss-2023a
```

**Pull request merged**